app/clients/zendesk_sell.py

```diff
@@ -249,6 +296,32 @@ def upsert_deal(self, contact_id: int, service: Service, stage_id: int) -> Optio
             current_app.logger.warning(f'Invalid response: {resp.text}')
             return None
 
+    def create_note(self,
+                    resource_type: NoteResourceType,
+                    resource_id: str,
+                    contact: ContactRequest) -> Optional[str]:
+
+        # The API and field definitions are defined here:
+        # https://developers.getbase.com/docs/rest/reference/notes
+        resp, e = self._send_request(
+            method='POST',
+            relative_url='/v2/notes',
+            data=json.dumps(
```

Comment:
I believe you can just pass a `json` arg to `requests`, don't know if your private method accepts that